source/_docs/pantheon-yml.md

```diff
@@ -78,6 +78,23 @@ remote: Valid versions are: 1
 
 While our parser will reject a `pantheon.yml` that is invalid, it won't necessarily give you the exact reason the file is invalid. Please refer to the examples above for exact syntax.
 
-### See Also
+### Deploying Configuration Changes to Multidev
+If a `pantheon.yml` file **does not** exist on the Dev environment, configuration changes will not be detected upon initial commit to a Multidev environment.
```

Comment:
@rachelwhitton Can we word this:

"If a `pantheon.yml` file **does not** exist on the Dev environment, configuration changes will not be detected when creating a multidev branch from an existing git branch that includes a `pantheon.yml` file."

@ari-gold If a `pantheon.yml` is pushed up to a mutlidev after the initial commit, still without a `pantheon.yml` in dev, will it be detected?